docs/guides/LocalLLMs.md

```diff
@@ -1,49 +1,40 @@
 # Local LLM Guide with Ollama server
 
-## 0. Install and Start ollama:
-run the following command in a conda env with CUDA etc.
+Ensure that you have the Ollama server up and running.
+For detailed startup instructions, refer to the [here](https://github.com/ollama/ollama)
 
-Linux:
-```
-curl -fsSL https://ollama.com/install.sh | sh
-```
-Windows or macOS:
+## 1. Pull Models
 
-- Download from [here](https://ollama.com/download/)
```

Comment:
I removed all this because I find the Ollama instructions clear and comprehensive, also providing various installation methods.